pkg/private/tar/build_tar.py

```diff
@@ -44,7 +44,7 @@ class DebError(Exception):
 
   def __init__(self, output, directory, compression, compressor, create_parents,
                allow_dups_from_deps, default_mtime, compression_level,
-               tar_format=None):
+               tar_format=None, allow_prefix_duplication=True):
```

Comment:
This should default to false so the default behavior is consistent with the mainline repo